<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewResponseDialog</class>
 <widget class="QDialog" name="NewResponseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelTaskClass">
         <property name="text">
          <string>Task Class</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Nutmeg::ComboBox" name="comboTaskClass">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelTask">
         <property name="text">
          <string>Task</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Nutmeg::ComboBox" name="comboTask">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelMatter">
         <property name="text">
          <string>Matter
(Can create new)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Nutmeg::MatterSearchBox" name="comboMatter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>488</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbDates">
     <property name="title">
      <string>Dates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelTriggerDate">
          <property name="text">
           <string>Trigger Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Nutmeg::DateEdit" name="deTriggerDate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="labelSoftDeadline">
          <property name="text">
           <string>Soft Deadline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Nutmeg::DateEdit" name="deSoftDeadline"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="labelHardDeadline">
          <property name="text">
           <string>Hard Deadline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Nutmeg::DateEdit" name="deHardDeadline"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labelNextDeadline">
          <property name="text">
           <string>Next Deadline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Nutmeg::DateEdit" name="deNextDeadline"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="gbFlags">
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QCheckBox" name="cbRequiresAuth">
        <property name="text">
         <string>Needs Authorization?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAuthReceived">
        <property name="text">
         <string>Authorization Received?</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>145</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Nutmeg::MatterSearchBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/mattersearchbox.h</header>
  </customwidget>
  <customwidget>
   <class>Nutmeg::DateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">widgets/dateedit.h</header>
  </customwidget>
  <customwidget>
   <class>Nutmeg::ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewResponseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewResponseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
